test/UnitTests/MSTest.CoreAdapter.Unit.Tests/Discovery/AssemblyEnumeratorTests.cs

```diff
@@ -262,7 +262,7 @@ public void EnumerateAssemblyShouldReturnTestElementsForAType()
 
         var testElements = testableAssemblyEnumerator.EnumerateAssembly("DummyAssembly", out _warnings);
 
-        VerifyCollectionsAreEqual(new Collection<UnitTestElement> { unitTestElement }, testElements.ToList());
+        Verify(new Collection<UnitTestElement> { unitTestElement }.SequenceEqual(testElements.ToList()));
```

Comment:
Nit: `SequenceEqual` working with `IEnumerable<T>` so virtually all collections.

```suggestion
        Verify(new Collection<UnitTestElement> { unitTestElement }.SequenceEqual(testElements));
```